playbooks/setup_jenkins_slave.yml

```diff
@@ -12,7 +12,6 @@
         - git
         - default-jre-headless
         - python-dev
-        - python-pip
         - build-essential
```

Comment:
``build-essential`` is not necessary, just ``gcc`` and ``libffi-dev`` are good enough